### PR TITLE
2nd password input field user settings

### LINKED ISSUE #488: Password change
When changing your password you should have 2 fields to put in the "new password".
A call to the backend should only be made when the 2 inputs for the new password are the same.

### PR DESCRIPTION
**Pull request by HuanYu-Tan**
fixes #488.

### CODE REVIEW

File: frontend/components/Settings/Settings.tsx [hunk 1]
```diff
@@ -79,6 +84,9 @@ export const Settings: React.FC<{
         } else if (currPassword !== "" && newPasswordScore < 20) {
             setNewPasswordError("Please provide a secure enough password");
             error = true;
+        } else if (newPassword !== retypePassword) {
+            setNewPasswordError("The provided passwords must be the same");
```
Comment (Mouwrice) [inline frontend/components/Settings/Settings.tsx:88]: Use the same message as from the login page to keep it consistent.

**Review by Mouwrice — CHANGES_REQUESTED** [review]
Nice addition.

File: frontend/components/Settings/Settings.tsx (hunk 1 above)
Comment (Mouwrice) [reply frontend/components/Settings/Settings.tsx:88]: Maybe place the error message in a field under the retypePassword field, as in the login page.

**Review by KoenDesplenter — APPROVED** [review]
Great

**Review by jay-tux — APPROVED** [review]
:rocket:

**Pull request merged**